app/src/ui/app.tsx

```diff
@@ -387,7 +387,9 @@ export class App extends React.Component<IAppProps, IAppState> {
       case 'view-repository-on-github':
         return this.viewRepositoryOnGitHub()
       case 'compare-on-github':
-        return this.compareBranchOnDotcom()
+        return this.openBranchOnGitub(true)
```

Comment:
Sorry for being picky but I would call out "compare=true/false" explicitly for clarity